docs/USAGE.md

```diff
@@ -111,6 +111,37 @@ You can find the scss files here:
 - dist/styles/variables - containing all the variables for colors, typography, etc
 - dist/lib - containing all the components styles
 
+### Canopy Fonts
+
+Canopy consists of two fonts; Roboto and Lyon. Roboto is included in the distributed Canopy package, and will load automatically when you have installed Canopy correctly.
+
+Due to our commericial license agreement, Lyon is not included in the Canopy distributed package, so you need to include it in the **assets** directory of your application. Canopy still references Lyon in the \`\`@font-face\`\` rulesets, so you don't need to implement them, but you do need to ensure that the path to the font in your assets directory is the same as the paths specified by Canopy in \`\`@font-face\`\`.
+
+Canopy's path for Lyon is **/assets/fonts/lyon/**, therefore you need to put the font files in the following location:
+
+```bash
+my-app/
+  src/
+    assets/
+      fonts/
+        lyon/
+          LyonDisplay-Regular-Web.woff
+          LyonDisplay-Regular-Web.woff2
+          LyonDisplay-Bold-Web.woff
+          LyonDisplay-Bold-Web.woff2
+```
+
+> This will result in the font being served in your app like this: <br>http://my-app.landg.com/assets/fonts/lyon/LyonDisplay-Regular-Web.woff2
+
+When you include the canopy.css file in your project, it will try to load the font from that exact path. If your font files are in the wrong location, or have a typo in the filename, the Lyon font won't load correctly.
+
+#### How is Lyon omitted? 
```

Comment:
I'm not sure we need this here? Users don't really have to know this 🤔

Reply:
I was wondering about this too. In the end I decided to include it to help future developers, should they need to alter the Webpack config or add another commercial font. So if we were to keep it in some way, perhaps it should go into CONTRIBUTING.md - or maybe just remove it entirely 🤔  What's your preference? I'm happy to go with the consensus on this (CC @thatguynamedandy, @paul-request and  @aaronBery)

Reply:
Maybe I'd move it to `CONTRIBUTING.md`

Reply:
I think it would be better in CONTRIBUTING.md, or I would actually favour putting it in the git commit message (which is personal preference more than anything)